exercises/practice/bowling/test/bowling_test.exs

```diff
@@ -103,6 +103,14 @@ defmodule BowlingTest do
     assert Bowling.score(game) == {:ok, 30}
   end
 
+  @tag :pending
+  test "efb426ec-7e15-42e6-9b96-b4fca3ec2359" do
```

Comment:
Can this be updated with a more descriptive title?

Reply:
```suggestion
  test "last two strikes followed by only last bonus with non strike points" do
```
Oops , copy/paste error.